OpenUpSA/wazimap-ng-ui
# closing the panels on map download

## Description
if there are open left panels, instead of disabling the map download button
* the open left panels will be closed when the user clicks map download button
* a title attribute will be added to the map download button that will state `The panes on the left will be closed to download the full-screen map`


## Related Issue
https://wazimap.atlassian.net/browse/WNCM-131

## How is it tested automatically?
added gui test `__tests__/gui/map_download.feature`

## How should a reviewer test it locally
* expand one of the panels on the left-hand side e.g. data mapper
* confirm that the map download button is not disabled
* hover over the map download button
* confirm that a tooltip that states `The panes on the left will be closed to download the full-screen map` is shown
* click on the map download button
* confirm that the open panel on the left-hand side is closed
* confirm that the downloaded image does not show the previously open panel on the left


## Screenshots
![image](https://user-images.githubusercontent.com/53019884/143193121-3143f4f5-59bb-42e6-ad7f-046baabcd122.png)


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
